TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -152,6 +156,21 @@ class BenchmarkParser extends Parser {
                 benchmarkProduct: curBenchmarkProduct,
                 testData: curTestData
             } );
+            // collect info and make calculation based on the curTestData
+            if ( curTestData.metrics && curTestData.metrics.length > 0 ) {
```

Comment:
We can just check if `curTestData.metrics` is an array:
```
if (Array.isArray(curTestData.metrics)) {
```

Reply:
Updated!